modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/index/AbstractSortedIndexStorageTest.java

```diff
@@ -308,8 +313,52 @@ void testScan() {
         }
     }
 
-    @Test
-    public void testBoundsAndOrder() {
+    /**
+     * Tests the happy case of the {@link SortedIndexStorage#readOnlyScan} method.
+     */
+    @RepeatedTest(5)
+    void testReadOnlyScan() {
```

Comment:
Looks like a copy of another test. Can you reuse the code instead of copying it?